### PR TITLE
arabic lang

### PR DESCRIPTION
**Pull request by Omaranwa**
hello ::
its arabic lang for the game i will update it as i can

### CODE REVIEW

**Comment by Gymnasiast** [comment]
ar-AR is not a valid code. It means 'Arabic from Argentina'.

I can look up the correct code for you. In which country do you live?

**Comment by Nubbie** [comment]
As I only see one string translated, maybe you could make some more commits and changes before it being merged :) 
(Just so it gets updated and not left at <5%)

**Comment by Omaranwa** [comment]
hello there i am from egypit. also as i sad  this is for testing only i will update it soon

**Comment by Gymnasiast** [comment]
Good!

Could you rename the file to `ar-EG.txt`?

**Comment by Omaranwa** [comment]
ok i will do

File: data/language/ar-ُEG.txt [hunk 1]
```diff
@@ -3731,1103 +3731,4 @@ STR_5389    :{SMALLFONT}{BLACK}Skip to the selected command in the title sequenc
...
-STR_NAME    :Condor Ride
-STR_DESC    :Riding in special harnesses below the track, riders experience the feeling of flight as they swoop through the air in Condor-shaped trains
-STR_CPTY    :4 passengers per car
-
-#WW
-[CONGAEEL]
-STR_NAME    :Conger Eel Coaster
-STR_DESC    :A compact steel-tracked roller coaster where the Eel-shaped train travels through corkscrews and loops
-STR_CPTY    :4 passengers per car
+STR_5393    :{SMALLFONT}{BLACK}Remove the selected save from the
```
Comment (Gymnasiast) [inline data/language/ar-ُEG.txt]: You cut off a large part of the file here. I think that was by accident.

Comment (Omaranwa) [inline data/language/ar-ُEG.txt]: sorry for that

**Pull request closed without merging**